OffchainLabs/arb-token-bridge
# chore: don't send sentry errors when user cancel metamask popup

### Summary

Ignore 4001 errors (cancellation from user) when switching network to avoid noise in sentry



### Steps to test

- Switch network and click on reject. Nothing should be sent to sentry
- Switch network with error (other than cancellation), error should be sent to sentry
